gui/console.cpp

```diff
@@ -646,8 +660,18 @@ void ConsoleDialog::historyScroll(int direction) {
 		_history[_historyIndex] = getUserInput();
 
 	// Advance to the next line in the history
+	// Note that due to temporarily storing the user input line in the _history table,
+	//      without executing an AddToHistory() call,
```

Comment:
```suggestion
	//      without executing an addToHistory() call,
```
That's really nitpicking though :-)

Reply:
Fixed in the new commit :) https://github.com/scummvm/scummvm/pull/3804/commits/f58aa17c4eead86f0116be320676aff5f5fc9c90